pkg/storage/certificates/private_key1.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Typically public keys should not be stored in the repo.

Reply:
The decision at the moment not to focus on key generation, saving to the database and its regeneration (it would be better to make a separate service to manage all this things).

One other way is to save all data as a constant value but it will not be possible to replace it during server runtime.

Reply:
You mean **private**. 

We decided to use the simplest way just to not complicate implementation with secure vault storage for now.